File: src/main/java/com/woomulwoomul/woomulwoomulbackend/domain/user/UserRoleRepository.java
package com.woomulwoomul.woomulwoomulbackend.domain.user;

import com.woomulwoomul.woomulwoomulbackend.domain.user.custom.UserRoleCustomRepository;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UserRoleRepository extends JpaRepository<UserRole, Long>, UserRoleCustomRepository {
}

File: src/main/kotlin/com/woomulwoomul/woomulwoomulbackend/domain/user/UserRoleRepository.kt
package com.woomulwoomul.woomulwoomulbackend.domain.user

import com.woomulwoomul.woomulwoomulbackend.domain.user.custom.UserRoleCustomRepository
import org.springframework.data.jpa.repository.JpaRepository

interface UserRoleRepository : JpaRepository<UserRoleEntity, Long>, UserRoleCustomRepository {
}